flytekit/clis/sdk_in_container/package.py

```diff
@@ -125,8 +126,7 @@ def package(ctx, image_config, source, output, force, fast, in_container_source_
                     archive_fname = os.path.join(output_tmpdir, f"{digest}.tar.gz")
```

Comment:
not your code, but i think compute_digest is wrong!. It is computing on the entire source and not the filtered objects only